frontend/app/work_packages/controllers/work-packages-controller.js

```diff
@@ -50,16 +50,12 @@ module.exports = function($scope, $state, $stateParams, QueryService, PathHelper
     return $state.is('work-packages.list');
   };
 
-  $scope.isShowViewActive = function() {
-    return $state.includes('work-packages.show');
-  };
-
   $scope.getToggleActionLabel = function(active) {
     return (active) ? I18n.t('js.label_deactivate') : I18n.t('js.label_activate');
   };
 
   $scope.getActivationActionLabel = function(activate) {
-    return (activate) ? I18n.t('js.label_activate') : '';
+    return (activate) ? I18n.t('js.label_activate') + ' ' : '';
```

Comment:
To reduce redundancy when using the method.
If the label is active, there should be a space after it, because it is always followed by another word.
If it wouldn't, the empty space is not visible in the HTML anyway.